.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Please remove the branches filter on pull requests.

Reply:
Will that change the behavior to run on every change to the repo?

Reply:
Without a filter it will run on PR made to other branches as well, see https://github.com/servicemeshinterface/smi-sdk-go/blob/master/.github/workflows/ci.yaml

Reply:
Removed as requested